data/blood/glucose/glucose.go

```diff
@@ -62,7 +63,9 @@ func NormalizeValueForUnits(value *float64, units *string) *float64 {
 	if value != nil && units != nil {
 		switch *units {
 		case MgdL, Mgdl:
-			return app.FloatAsPointer(*value / MmolLToMgdLConversionFactor)
+			intValue := int(*value/MmolLToMgdLConversionFactor*MmolLToMgdLPrecisionFactor + 0.5)
```

Comment:
It is just to force the integer value to round (rather than floor). Do you think it needs a constant?